R/utils-classes.R

```diff
@@ -231,6 +231,33 @@ select.bcdc_promise <- function(.data, ...){
 }
 
 
+#' Mutate columns from Web Service call
```

Comment:
I feel like the title should be more reflective of what the function does (not what  user hopes it will do) - thoughts?

```suggestion
#' Throw an informative error when attempting mutate on a Web Service call
```